### PR TITLE
Fixed ~300 weight class errors

### PR DESCRIPTION
**Pull request by photon-rain**

### CODE REVIEW

File: meet-data/apa/1711/entries.csv [hunk 1]
```diff
@@ -19,13 +19,13 @@ Ken Morgan,MS,SBD,M,Master I.,Wraps,124.38,125,42,306.17,213.19,292.57,811.93,1
...
-Chris Pereira,MS,SBD,M,Junior (by Formula),Wraps,88.9,90,20,258.55,167.83,265.35,691.73,3
-Jake Shelton,MS,SBD,M,Junior (by Formula),Wraps,139.34,140,22,299.37,183.7,292.57,775.64,4
-Tony Beckham,MS,SBD,M,Junior (by Formula),Wraps,60.06,60,20,142.88,120.2,174.63,437.72,5
-Joshua Hayes,FL,SBD,M,Junior (by Formula),Wraps,99.34,100,20,215.46,138.35,240.4,594.21,6
-Eric Elsey,MS,SBD,M,Junior (by Formula),Wraps,69.31,75,21,,,,,DQ
+Cameron Moss,AL,SBD,M,Junior,Wraps,117.66,125,20,276.69,199.58,331.12,807.39,1
+Damien Seales,MS,SBD,M,Junior,Wraps,65.14,67.5,20,208.65,113.4,247.21,569.26,2
+Chris Pereira,MS,SBD,M,Junior,Wraps,88.9,90,20,258.55,167.83,265.35,691.73,3
+Jake Shelton,MS,SBD,M,Junior,Wraps,139.34,140,22,299.37,183.7,292.57,775.64,4
+Tony Beckham,MS,SBD,M,Junior,Wraps,60.06,67.5,20,142.88,120.2,174.63,437.72,5
```
Comment (sstangl) [inline meet-data/apa/1711/entries.csv]: This one should just have bodyweight changed to 60 -- they meant to enter him as "maximum allowable by weightclass" but then got caught by kg/lbs rounding.

File: meet-data/spf-archive/0115/entries.csv [hunk 2]
```diff
@@ -25,7 +25,7 @@ Augie,99.34,100,M,Multi-ply,Amateur Open,335.66,,,335.66,208.65,222.26,231.33,23
 Howard Murphy,97.52,100,M,Multi-ply,Amateur Open,297.1,308.44,,308.44,165.56,174.63,181.44,181.44,256.28,272.16,,272.16,762.04,SBD,2
 Jimmie Pacifico,98.88,100,M,Multi-ply,Pro Open,385.55,,,385.55,240.4,,256.28,256.28,272.16,290.3,,290.3,932.13,SBD,1
 Rob McNutt,99.79,100,M,Multi-ply,Pro Open,,308.44,317.51,317.51,,242.67,244.94,244.94,265.35,,,265.35,827.8,SBD,2
-Matt Adams,100.02,100,M,Multi-ply,Pro Open,267.62,281.23,,281.23,226.8,,,226.8,249.48,276.69,,276.69,784.72,SBD,3
+Matt Adams,100.02,110,M,Multi-ply,Pro Open,267.62,281.23,,281.23,226.8,,,226.8,249.48,276.69,,276.69,784.72,SBD,3
```
Comment (sstangl) [inline meet-data/spf-archive/0115/entries.csv]: Same.

File: meet-data/spf-archive/0110/entries.csv [hunk 3]
```diff
@@ -29,7 +29,7 @@ Jeremy Hannah,102.51,AL,,197.31,,197.31,,,B,Raw,M,Juniors,110,1
 Kim Craft,56,AL,,61.23,,61.23,,,B,Raw,F,Submasters,56,1
 Greg Hickman,67.13,AL,,120.2,,120.2,,,B,Raw,M,Juniors,67.5,1
 Lee Morris,89.99,AL,,183.7,,183.7,,,B,Raw,M,Masters 40-44,90,1
-John Burns,117.57,TN,,276.69,,276.69,,,B,Multi-ply,M,Open,117.5,1
+John Burns,117.57,TN,,276.69,,276.69,,,B,Multi-ply,M,Open,125,1
```
Comment (sstangl) [inline meet-data/spf-archive/0110/entries.csv]: Same.

File: meet-data/spf-archive/0497/entries.csv [hunk 4]
```diff
@@ -51,7 +51,7 @@ Zachary Mullins,TX,33,M,Wraps,Open,125,123.79,250,185,257.5,692.5,SBD,1
 Alan Charlton,TX,24,M,Wraps,Open,140,138.39,240,137.5,240,617.5,SBD,1
 Adam Stephens,TX,24,M,Wraps,Open,140+,169.92,265,170,320,755,SBD,1
 Rob Williamson,TX,46,M,Raw,Masters 1,82.5,79.56,97.5,80,137.5,315,SBD,1
-Scott Friend,TX,29,M,Raw,Open,100,100.02,162.5,117.5,185,465,SBD,2
+Scott Friend,TX,29,M,Raw,Open,110,100.02,162.5,117.5,185,465,SBD,2
```
Comment (sstangl) [inline meet-data/spf-archive/0497/entries.csv]: Here too.

File: meet-data/spf-archive/0480/entries.csv [hunk 5]
```diff
@@ -16,12 +16,12 @@ Shawna Mendelson,NY,43,80.74,82.5,F,Multi-ply,Open,317.51,188.24,219.99,725.75,S
 Bruce Maculan,RI,70,85.73,90,M,Raw,Masters 70+,97.52,77.11,154.22,328.85,SBD,1
 Brian Wambolt,RI,38,88.9,90,M,Multi-ply,Open,283.5,265.35,242.67,791.52,SBD,2
 Nelson Snyder,PA,38,89.13,90,M,Wraps,Submaster,244.94,176.9,263.08,684.92,SBD,1
-David Kirschen,NJ,38,90.04,90,M,Multi-ply,Open,367.41,272.16,260.82,900.38,SBD,1
+David Kirschen,NJ,38,90.04,100,M,Multi-ply,Open,367.41,272.16,260.82,900.38,SBD,1
```
Comment (sstangl) [inline meet-data/spf-archive/0480/entries.csv]: Here.

File: meet-data/spf-archive/0480/entries.csv [hunk 6]
```diff
@@ -16,12 +16,12 @@ Shawna Mendelson,NY,43,80.74,82.5,F,Multi-ply,Open,317.51,188.24,219.99,725.75,S
...
 Ethan Ostermeyer,RI,25,95.25,100,M,Raw,Open,,183.7,,183.7,B,2
 Nick Blundi,NJ,28,95.71,100,M,Raw,Open,,188.24,,188.24,B,1
 Nate Lawrence,NH,23,96.62,100,M,Multi-ply,Open,294.84,210.92,272.16,777.91,SBD,1
 Michael Bove,RI,34,99.11,100,M,Wraps,Open,181.44,136.08,204.12,521.63,SBD,2
-Anthony Buono,RI,33,100.02,100,M,Wraps,Open,181.44,122.47,235.87,539.77,SBD,1
+Anthony Buono,RI,33,100.02,110,M,Wraps,Open,181.44,122.47,235.87,539.77,SBD,1
```
Comment (sstangl) [inline meet-data/spf-archive/0480/entries.csv]: And here.

File: meet-data/spf/1730/entries.csv [hunk 7]
```diff
@@ -1,7 +1,7 @@
 Name,Division,Equipment,WeightClassKg,BodyweightKg,Event,BestSquatKg,BestBenchKg,BestDeadliftKg,TotalKg,Sex,Place
 Adrian Ramos,Sub-Masters,Wraps,117.5,117.03,SBD,247.66,160.12,262.18,669.96,M,1
-AJ Alvarez,Juniors,Wraps,100,100.02,SBD,307.54,200.03,317.51,825.08,M,1
-AJ Alvarez,Open,Wraps,100,100.02,SBD,307.54,200.03,317.51,825.08,M,2
+AJ Alvarez,Juniors,Wraps,110,100.02,SBD,307.54,200.03,317.51,825.08,M,1
+AJ Alvarez,Open,Wraps,110,100.02,SBD,307.54,200.03,317.51,825.08,M,2
```
Comment (sstangl) [inline meet-data/spf/1730/entries.csv]: These two entries also.

**Comment by photon-rain** [comment]
@sstangl Is this ok to merge now? I fixed all the pound -> kg conversion errors.

**Pull request merged**